dev/NavigationView/TestUI/Common/NavigationViewPage.xaml

```diff
@@ -282,6 +282,16 @@
                     <StackPanel Orientation="Horizontal" Margin="0 5">
                         <Button x:Name="BringSettingsIntoViewButton" AutomationProperties.Name="BringSettingsIntoViewButton" Click="BringSettingsIntoViewButton_Click"
                                 Content="Bring Settings into view"/>
+                        <ComboBox x:Name="ScrollItemIntoViewComboBox"
```

Comment:
The `BringSettingsIntoViewButton` ui element is now a relict since PR https://github.com/microsoft/microsoft-ui-xaml/pull/3602 restored the Settings item's pinned viewport behavior. This button can now be removed in the test UI, however, as multiple unrelated interaction tests are using it, I would suggest that if we remove that button, we can do that in a follow-up PR. Let's first bring this PR over the finish line so it won't block the WinUI 2.5 release.

(Alternatively, the functionality of this button could be integrated into the newly introduced ScrollItemIntoViewComboBox as part of this PR, though it's not required for now.)